src/sagas/index.js

```diff
@@ -657,31 +660,46 @@ export function* stopProgress ({ vmId, poolId, name, result }) {
 function* shutdownVm (action) {
   yield startProgress({ vmId: action.payload.vmId, name: 'shutdown' })
   const result = yield callExternalAction('shutdown', Api.shutdown, action)
+  if (result.status === 'complete') {
+    yield put(addUserMessage({ message: msg.actionFeedbackShutdownVm({ VmId: action.payload.vmId }), type: 'success' }))
```

Comment:
Looking good, almost there.  Using the VM Name would be more appropriate here.  The only place I remember seeing the VM id itself is in the URL for the details page.

In a saga, given the VM id, you can get the VM name like this:
```jsx
const vmName = yield select(state => state.vms.getIn([ 'vms', vmId, 'name' ]))
```